oks767/main.js

```diff
@@ -0,0 +1,137 @@
+import data from './tasks.json' assert { type: 'json' };
+let addTaskForm = document.querySelector('.form');
+let addTaskFormLow = document.querySelector('.form-low');
+let inputHigh = document.getElementById('todo-high');
+let inputLow = document.getElementById('todo-low');
+let todoList = document.querySelector('.todo-list');
+let todoListLow = document.querySelector('.todo-list-low');
+
+//todo array
+const STATUSES = {
+  TODO: 'to do',
+  DONE: 'done',
+};
+
+const PRIORITY = {
+  HIGH: 'high',
+  LOW: 'low',
+};
+
+const list = [];
+document.addEventListener('DOMContentLoaded', () => {
+  list.push(data);
+  list.forEach(
+    data.forEach((item) => {
```

Comment:
```suggestion
  list.push(data);
  list.forEach(
    data.forEach((item) => {
```
:22 Пушишь массив в массив.
:23 В forEach не передан колбэк и похоже он здесь вообще лишний
:24 Работаешь с data,  a не list 
Я бы вынес отрисовку элементов в функцию render()